### ISSUE TITLE
Display tooltip based on mouse movement

### ISSUE THREAD

**Issue by komelon** [Live-Charts/Live-Charts#1078]
[https://imgur.com/07HcKt8](url)
[https://imgur.com/Cr04gn5](url)

I want the tooltip to display the point value according to the mouse cursor movement, as in the image link above.
I tried, but I haven't found a way to display the tooltip without hovering the mouse cursor over the point in Live-Charts.

If anyone has done this, can you give some advice?

**Comment by yesyesuk** [comment]
Yes, this is something I would like too. The tooltip should depend on the X position of the mouse pointer, but not on the Y position, as long as the pointer is within the chart area.

**Comment by komelon** [comment]
[Stackoverflow Answer Link](https://stackoverflow.com/questions/62714946/wpf-live-charts-display-tooltip-based-on-mouse-cursor-move-without-mouse-hover-o/62743437#62743437)

I asked Stackoverflow the same question and 'BionicCode' gave a precious answer.
Thanks to him and leave a link for users who use wpf-charts.

**Comment by kMOsmane** [comment]
> 
> 
> [Image Link 1] (https://imgur.com/07HcKt8) [Image Link 2] (https://imgur.com/Cr04gn5)
> 
> I want the tooltip to display the point value according to the mouse cursor movement, as in the image link above. I tried, but I haven't found a way to display the tooltip without hovering the mouse cursor over the point in Live-Charts.
> 
> If anyone has done this, can you give some advice?

did you find the solution of your idea ?
